### ISSUE TITLE
TODO make the program be only one window.

### ISSUE THREAD

**Issue by Kainny2000** [alex0112/abacus#124]
Modify app so it doesn't have the first two intro screens so that everything is just on one main window (Welcome UVsim and Load File).
 TODO feedback: "Generally, your app still does not benefit from having multiple windows/frames and moving between them."..."On the new stuff, the color scheme works fine, although (per the comment above) having the color scheme available on the 'intro' window only and not on the main window is inconvenient since the user can't go back to it without restarting the app. After running a file and the user wants to run a new file, the second version of the file picker at the bottom of the main frame does not work (doesn't load anything) -- again I have to restart the app and use the 'intro' file picker to load a file correctly (another issue with having multiple frames if the codebase is different between these two file pickers and aren't synced or tested properly with everything else).
 
 **Looks like Jackson worked on moving the changing color, waiting on the pull request
